components/reusable-components/navigation/navigation.module.scss

```diff
@@ -0,0 +1,126 @@
+@mixin respond-to($size) {
+  @if $size == 'small' {
+    @media (max-width: 768px) {
+      @content;
+    }
+  } @else if $size == 'medium' {
+    @media (min-width: 769px) and (max-width: 1024px) {
+      @content;
+    }
+  } @else if $size == 'large' {
+    @media (min-width: 1025px) and (max-width: 1440px) {
+      @content;
+    }
+  } @else if $size == 'extra-large' {
+    @media (min-width: 1441px) {
+      @content;
+    }
+  }
+}
+
+.navbar {
+  background-color: var(--bg-color);
+}
+
+.navbarBrand {
+  color: var(--text-color);
+  margin-left: 0.55em;
+  width: 6.3em;
+  height: 1.22em;
+  top: 1.86em;
+  left: 2.36em;
+}
+
+.navLink1 {
```

Comment:
Not descriptive enough, the link can be changed in the future, would the same styles apply?